### DISCUSSION TITLE
Guide

### DISCUSSION THREAD

**Discussion by boxadesign** [ONSdigital/design-system#1401] [Feedback]
Use this place to discuss the [guide pattern](https://ons-design-system.netlify.com/patterns/guide) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Guide and RTL examples showing pager arrow icons in wrong directions
![Screenshot 2021-10-19 at 11 31 17](https://user-images.githubusercontent.com/43346934/137892625-7c6aca1f-da96-4146-ab14-f40cf7919925.png)

**Reply by jrbarnes9** [reply]
#1734 resolved
